docs/man_pages/project/testing/build-android.md

```diff
@@ -27,10 +27,11 @@ General | `$ tns build android [--compileSdk <API Level>] [--key-store-path <Fil
 * `--copy-to` - Specifies the file path where the built `.apk` will be copied. If it points to a non-existent directory path, it will be created. If the specified value is existing directory, the original file name will be used.
 * `--env.*` - Specifies additional flags that the bundler may process. May be passed multiple times. Supported additional flags:
     *   `--env.aot` - creates Ahead-Of-Time build (Angular only).
-    *   `--env.snapshot`- creates [Snapshot](https://docs.nativescript.org/performance-optimizations/bundling-with-webpack#v8-heap-snapshot) (only for release builds on Mac OS & for Android).
+    *   `--env.snapshot`- creates [a V8 Snapshot](https://docs.nativescript.org/performance-optimizations/bundling-with-webpack#v8-heap-snapshot) decreasing the app start time (only for release builds for Android).
+    *   `--env.compileSnapshot`- compiles the static assets produced by `--env.snapshot` into `.so` files allowing the native build to split them per architecture. In this way, the app size will be reduced when using the `--aab` option. 
```

Comment:
` In this way, the app size will be reduced` - `This will reduce the app size...`